### PR TITLE
Update MAINTAINERS.md

### PR DESCRIPTION
**Pull request by jmmut**

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ Past CRAM maintainers include Vadim Zalunin.
 
 ### VCF/BCF
 
-* Cristina Yenyxe Gonzalez Garcia (@cyenyxe)
+* Jose Miguel Mut Lopez (@jmmut)
 * Louis Bergelson (@lbergelson)
 * Petr Danecek (@pd3)
 
-Past VCF/BCF maintainers include Ryan Poplin and David Roazen.
+Past VCF/BCF maintainers include Ryan Poplin, David Roazen and Cristina Yenyxe Gonzalez Garcia.
```
Comment (jmarshall) [inline MAINTAINERS.md]:
```suggestion
Past VCF/BCF maintainers include Ryan Poplin, David Roazen, and Cristina Yenyxe Gonzalez Garcia.
```

For the record, does @cyenyxe sort herself (by surname) at G or Y?

Comment (jmmut) [reply MAINTAINERS.md]: I was not aware of any name ordering in this page, should I change this?

Sorting by surname 'Gonzalez' should be used.

Comment (jmarshall) [reply MAINTAINERS.md]: Whether the current maintainers are sorted, and if so how, is an open question (cf #415). (If it were me, I would add myself at the bottom, but that's just MHO.)

The lists of past maintainers are **alphabetical** (and use Oxford commas).

**Comment by jmarshall** [comment]
Changes to the listed maintainers also require an expression of approval by @ohofmann and/or @tk2.

**Comment by ohofmann** [comment]
Where do I sign... ? Go for it.

**Comment by tk2** [comment]
LGTM 👍

**Pull request merged**